### PR TITLE
Desktop portfolio version

### PR DESCRIPTION
**Pull request by redwing555**
This PR is the desktop version of my personal portfolio webpage:

⏩ Creation of a media query for the desktop layout while respecting HTML/CSS best practices.
⏩Use of flexbox grid CSS for the layout.
⏩ Respect of Figma design as well as UX design best practices.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @redwing555,

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -435,8 +468,161 @@ footer {
...
+    background-image: url("img/background_aboutme.svg");
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .resume-tech {
+    display: flex;
+  }
+
+  .coding-languages,
```
Comment (mke2111) [inline style.css:562]: - [ ] Ensure that padding/margin on the left and right on all sections are equal.
- 'My recent works' has perfect padding/margin on the sides. Do the same for the technology section and the contact section.
Screenshot from 2021-08-20 11-57-14

**Review by Murodjon000 — CHANGES_REQUESTED** [review]
Hi Houssam Hichami 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] It would be better if you stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design because there are some issues in your design. The design you did is this.

- As you see, your containers are not aligned well with themselves. Such as the `projects container` is wider than the `about me` section container. If you look at the Figma design they should be aligned well on both left and right sides.
- Also the language lists inside the `projects container` are having issues with the text. Ruby on Rails text is overlapped. Please fix this also.
- Lastly, your `Form` section also should be aligned with your containers. Small hint they all should be inside one `main` tag and try to give them `max-width` to the container. The inputs also overflowed to the right side. Kindly try to make the design as the on [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)🙏.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @redwing555  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**